satellite/marketing/marketingweb/server.go

```diff
@@ -186,11 +190,23 @@ func formToStruct(w http.ResponseWriter, req *http.Request) (o marketing.NewOffe
 	return o,nil
 }
 
+func (s *Server) reRenderTables(w http.ResponseWriter){
+    req,err := http.NewRequest("GET","/",nil)
```

Comment:
Shouldn't `Redirect` already making a request to `/`?

Reply:
This will be the same user who made the `POST` request tho. Does it matter if we carry it over?

Reply:
For our use-case it doesn't really matter since it's behind a VPN but I think its a good practice in general for security.

Reply:
There's no difference security wise.

Reply:
Ok I'll get rid of it then.